tests/callbacks/test_callbacks.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 MosaicML LLM Foundry authors
+# SPDX-License-Identifier: Apache-2.0
+
+from composer.core import Callback
+
+from llmfoundry.registry import callbacks, callbacks_with_config
+from llmfoundry.utils.builders import build_callback
+
+
+def test_build_callbacks():
+    entry_points = callbacks.get_entry_points()
```

Comment:
I think you want to use this https://github.com/mosaicml/llm-foundry/blob/23caea7532fb37f23fde2a1bfff798f565f9aeb2/llmfoundry/utils/registry_utils.py#L69?

Reply:
oops, yeah you're right. What are entry points?

Reply:
entry points are just the part of the registry specified via pyproject.toml